symfony/templates/index.html.twig

```diff
@@ -86,16 +86,17 @@
     <script>
         $(document).ready(function () {
             // Add badge for count of Deceased Participant Reports for review
-            var deceasedReportCount = 0;
-            $.ajax({
-                url: '/s/deceased-reports/stats'
-            }).done(function (data) {
-                deceasedReportCount = data.pending
-                if (deceasedReportCount > 0) {
-                    $('#deceased_report_block > a').prepend($('<div class="welcome-page-badge">' + deceasedReportCount +'</div>'));
-                }
-            })
+            if ($('deceased_report_block').length) {
```

Comment:
Are you selecting id `deceased_report_block`? if yes it's missing id selector #

Reply:
Yep. 🤦‍♂️